src/transformers/models/ctrl/modeling_ctrl.py

```diff
@@ -575,6 +577,23 @@ def forward(
             attentions=transformer_outputs.attentions,
         )
 
+    @staticmethod
+    def _reorder_cache(past: Tuple[Tuple[torch.Tensor]], beam_idx: torch.Tensor) -> Tuple[Tuple[torch.Tensor]]:
+        """
+        This function is used to re-order the :obj:`past_key_values` or :obj:`mems` cache if
+        :meth:`~transformers.PretrainedModel.beam_search` or :meth:`~transformers.PretrainedModel.beam_sample` is
+        called. This is required to match :obj:`past_key_values` or :obj:`mems` with the correct beam_idx at every
+        generation step.
+
+        For custom re-ordering of :obj:`past_key_values` or :obj:`mems`, the function should be implemented in
```

Comment:
Yeah it would be nice if you could in a final step clean up the doc string here removing those two lines and removing all  mentioning of mems above

Reply:
Thanks! Referring to your comments, I've just done the clean up of the doc strings.